metcalcpy/util/read_file.py

```diff
@@ -0,0 +1,110 @@
+"""
+Program Name: read_file.py
+Copyright 2021 UCAR/NCAR/RAL
+"""
+
+__author__ = 'Hank Fisher'
+__version__ = '1.0.0'
+__email__ = 'met_help@ucar.edu'
+
+
+import numpy as np
+import pandas as pd
+import yaml
+from read_env_vars_in_config import parse_config
+#Two PYTHONPATH directories need to be set for the imports to work
+#The parent directory for where METdatadb is installed and
+#The METdbload/METdatadb/ush directory for constants in METdbload
+from METdatadb import METdbLoad as dbload
+from METdatadb.METdbLoad.ush import read_data_files
+from METdatadb.METdbLoad.ush.read_load_xml import XmlLoadFile
+
+class ReadMETOutput:
+
+    def __init__(self):
+        """ Creates a output reader with a list of files to read
+        """
+        self.flags = {}
+        self.flags['line_type_load'] = False
+        self.flags['load_stat'] = True
+        self.flags['load_mode'] = True
+        self.flags['load_mtd'] = True
+        self.flags['load_mpr'] = False
+        self.flags['load_orank'] = False
+        self.flags['force_dup_file'] = False
+        self.flags['verbose'] = False
+        self.flags['stat_header_db_check'] = True
+        self.flags['tcst_header_db_check'] = True
+        self.flags['mode_header_db_check'] = True
+        self.flags['mtd_header_db_check'] = True
+        self.flags['drop_indexes'] = False
+        self.flags['apply_indexes'] = False
+        self.flags['load_xml'] = True
+
+    def readYAMLConfig(self,configFile):
+        """ Returns a file or list of files
+        """
```

Comment:
need the Pydoc for the input var configFile:

@param configFile: some description here...